snap7/client.py

```diff
@@ -624,3 +627,58 @@ def set_plc_datetime(self, dt):
         buffer[5] = dt.year - 1900
 
         return self.library.Cli_SetPlcDateTime(self.pointer, byref(buffer))
+
+
+class ClientAsync(Client):
```

Comment:
It's a matter of taste. I put it in the same file, so that other developer may work easier, because the `ClientAsync` is only overwriting a few methods at the moment. As well an async Client would be still a Client for me.